test/import.cooltt

```diff
@@ -0,0 +1,13 @@
+import prelude
+  [ unit -> u /- rename unit to u -/
+  ; {-> p1, -> p2} /- union of two prefix modifiers -/
+  ; p2.[tt -> q.t] /- rename p2.tt to p2.q.t -/
+  ; [[[[[{[]};{[{[[];[]]}]}]]]]] /- no op -/
+  ; p1.[ ! u ] /- remove p1.u -/
```

Comment:
I wonder if we can instead use notations that are a little more clear for non-🧞. For instance:

1. Instead of `p.[...]` we might write `under p { ... }` or something
2. Instead of `! p` we might write `delete p`.

I don't feel strongly about it, but it might help.

Reply:
How about `under p [...]`? Currently, `{}` is for unions and `[]` is for sequencing, unless you would like to change that.

My only concern is that perhaps we would want to use `under` or `delete` in future cooltt code, but otherwise, yes we can do it...

Reply:
OK, let's not add these keywords.